_opensearch/search/autocomplete.md

```diff
@@ -0,0 +1,1032 @@
+            }
+          },
+          {
+            "text" : "To be a make-peace shall become my age:",
+            "_index" : "shakespeare",
+            "_id" : "78566",
+            "_score" : 1.0,
+            "_source" : {
+              "type" : "line",
+              "line_id" : 78567,
+              "play_name" : "Richard II",
+              "speech_number" : 20,
+              "line_number" : "1.1.160",
+              "speaker" : "JOHN OF GAUNT",
+              "text_entry" : "To be a make-peace shall become my age:"
+            }
+          },
+          {
+            "text" : "To be a party in this injury.",
+            "_index" : "shakespeare",
+            "_id" : "75259",
+            "_score" : 1.0,
+            "_source" : {
+              "type" : "line",
+              "line_id" : 75260,
+              "play_name" : "Othello",
+              "speech_number" : 57,
+              "line_number" : "5.1.93",
+              "speaker" : "IAGO",
+              "text_entry" : "To be a party in this injury."
+            }
+          },
+          {
+            "text" : "To be a preparation gainst the Polack;",
+            "_index" : "shakespeare",
+            "_id" : "33591",
+            "_score" : 1.0,
+            "_source" : {
+              "type" : "line",
+              "line_id" : 33592,
+              "play_name" : "Hamlet",
+              "speech_number" : 17,
+              "line_number" : "2.2.67",
+              "speaker" : "VOLTIMAND",
+              "text_entry" : "To be a preparation gainst the Polack;"
+            }
+          },
+          {
+            "text" : "To be a public spectacle to all:",
+            "_index" : "shakespeare",
+            "_id" : "3709",
+            "_score" : 1.0,
+            "_source" : {
+              "type" : "line",
+              "line_id" : 3710,
+              "play_name" : "Henry VI Part 1",
+              "speech_number" : 6,
+              "line_number" : "1.4.41",
+              "speaker" : "TALBOT",
+              "text_entry" : "To be a public spectacle to all:"
+            }
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+To specify the number of suggestions that you want to return, use the `size` parameter:
+
+```json
+GET shakespeare/_search
+{
+  "suggest": {
+    "autocomplete": {
+      "prefix": "To m",
+      "completion": {
+        "field": "text_entry",
+        "size": 3
+      }
+    }
+  }
+}
+```
+
+The maximum of three documents is returned:
+
+```json
+{
+  "took" : 1,
+  "timed_out" : false,
+  "_shards" : {
+    "total" : 1,
+    "successful" : 1,
+    "skipped" : 0,
+    "failed" : 0
+  },
+  "hits" : {
+    "total" : {
+      "value" : 0,
+      "relation" : "eq"
+    },
+    "max_score" : null,
+    "hits" : [ ]
+  },
+  "suggest" : {
+    "autocomplete" : [
+      {
+        "text" : "To m",
+        "offset" : 0,
+        "length" : 4,
+        "options" : [
+          {
+            "text" : "To make a bastard and a slave of me!",
```

Comment:
Small nit here, should we have harsh language in our example? I don't care personally but someone else might. :)

Reply:
definitely remove both words, those are on the [Offensive terms](https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md#offensive-terms) list. It says "don't use."

Reply:
Removed